frontend/prebuild/src/app/game/leaderboard/leaderboard.component.scss

```diff
@@ -0,0 +1,33 @@
+:host {
+  display: grid;
+  grid-template-rows: repeat(2, 1fr);
+  grid-template-columns: 50px 1fr;
+  min-height: 100%;
+  align-content: center;
+}
+
+#title {
+  grid-area: 1 / 1 / span 2 / 1;
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+}
+
+#title h2 {
+  transform: rotate(-90deg);
+  color: white;
+
+  text-transform: lowercase;
+  font-variant: small-caps;
+
+  letter-spacing: .1em;
+
+  margin: 0;
+  padding: 0;
+}
+
+.player-container {
```

Comment:
Did you mean `.leaderboard-container`? I don't see a player-container anywhere within the component...

Reply:
Good catch, I also realized that I'm not using any of the `ranking.component.*` files so I've removed those as well.  They were essentially replaced with this:
```html
<tr *ngFor="let ranking of rankings">
	<td>{{ ranking.rank }}</td>
	<td>{{ ranking.name }}</td>
	<td>{{ ranking.numWins }}</td>
	<td>{{ ranking.totalGames }}</td>
	<td>{{ ranking.winLossRatio }} %</td>
	<td>{{ ranking.rating }}</td>
</tr>
```
when I couldn't figure out how to make a proper `<ranking>` component